docs/src/styling.md

```diff
@@ -0,0 +1,112 @@
+other applications, and as such it requires its own guide to keep colours
+consistent. Towards this goal we will define several common types of
+applications and how to style each of them using the available colours.
+
+Please keep in mind that this is a general guide; there will be several
+applications that don't fit into any of the groups below. In this case it is up
+to the committer to make sure said application fits in stylistically with the
+rest of the themed applications.
+
+### Terms
+- Alternate: To use when to items need to contrast despite being the same type of item.
+- Item On/Off: This is for on off indicators in applications like a battery charging indicator.
+- List: A list of items to select between such as workspaces in a desktop bar.
+
+## General Applications
+
+General applications are applications that do not fit into any of the more
+specific categories below. This is the scheme you should default to if you are
+unsure. Examples include zathura and sxiv.
+
+- Default Background: base00 
+- Alternate Background: base01
+- Default Text Color: base05
+- Alternate Text Color: base04
+- Selection Color: base03
+- Urgent: base07
+- Error: base0F
+
+## Window Managers
+
+Window Managers arrange windows and provide decorations like title bars and
+borders. Examples include Sway and i3.
+
+This does not include applications bundled with the desktop environment such as
+file managers, which would fall into the general category.
+
+It also does not include desktop helpers such as taskbars and menus. However,
+these are often configured together with the window manager, especially in the
+case of larger desktop environments such as KDE and GNOME.
+
+- Unfocused window border: base03
+- Focused window border: base0A
+- Urgent window border: base07
+- Window title text: base05
+- Error: base0F
+
+
+## Notifications and Popups
+
+Notifications and popups are any application overlay intended to be displayed
+over other applications. Examples include the mako notification daemon and
+avizo.
+
+- Default Background Color: base00
+- Alternate Background: base01
+- Progress/Percentage Bar Color: base02
+- Alternate Text Color: base04
+- Default Text Color: base05
+- low Urgency Background Color: base06
+- High Urgency Text Color: base08
+- Low Urgency Text Color: base0A
+- Border Color: base0D
+- High Urgency Background Color: base0F
+
+## Desktop Helpers
+
+Applications that fall under this group are applications that complement the
+window management facilities of whatever window manager the user is using.
+Examples of this include waybar and polybar, as well as the similar programs
+that are part of KDE and GNOME.
+
+- Default Background Color: base00
+- Alternate Background Color: base01
+- Default Text Color: base05
+- Alternate Text Color: base04
+- Item On: base0E
+- Item Off: base0D
+- Alternate Item On: base09
+- Alternate Item Off: base02
+- List Background: base0D
+- Selected Item In List Background: base03
```

Comment:
`base0D` might make the text hard to read because it doesn't necessarily contrast with `base05`.

The GTK theme currently uses `base01` for certain lists:

![GNOME settings](https://github.com/danth/stylix/assets/28959268/b7d92a69-8a69-42f4-aed7-6e10715a2e84)

What do you think about using `base01` for the background and `base02` for the selected item?

Reply:
perhaps we should make a colored and uncolored section were in the colored section the text color is set to base00, kind of like the waybar module i made?